jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/DEF.java

```diff
@@ -22,74 +22,110 @@
-                                                                          contentTypeNQuadsAlt1
-                                                                          );
-
-    public static final AcceptList rdfOffer           = AcceptList.create(contentTypeTurtle,
-                                                                          contentTypeTurtleAlt1,
-                                                                          contentTypeNTriples,
-                                                                          contentTypeNTriplesAlt,
-                                                                          contentTypeRDFXML,
-                                                                          contentTypeTriX,
-                                                                          contentTypeTriXxml,
-                                                                          contentTypeJSONLD,
-                                                                          contentTypeRDFJSON,
-                                                                          contentTypeRDFThrift
-                                                                          );
-
-    public static final AcceptList quadsOffer         = AcceptList.create(contentTypeTriG,
-                                                                          contentTypeTriGAlt1,
-                                                                          contentTypeJSONLD,
-                                                                          contentTypeNQuads,
-                                                                          contentTypeNQuadsAlt1,
-                                                                          contentTypeTriX,
-                                                                          contentTypeTriXxml
-                                                                          );
+    // @formatter:off
+
+    // ---- Server configuration default media type
+
+    public static MediaType acceptRDFXML        = acceptRDFXMLDefault();
+    public static MediaType acceptNQuads        = acceptNQuadsDefault();
+    public static MediaType acceptResultSetXML  = acceptResultSetXMLDefault();
+    public static MediaType acceptJSON          = acceptJSONDefault();
+    public static MediaType acceptTurtle        = acceptTurtleDefault();
+
+    // ---- Server configuration offers for content negotiation
+
+    public static AcceptList jsonOffer          = jsonOfferDefault();
+
+    public static AcceptList constructOffer     = constructOfferDefault();
+
+    public static AcceptList rdfOffer           = rdfOfferDefault();
+
+    public static AcceptList quadsOffer         = quadsOfferDefault();
+
+    // Offer for SELECT
+    public static AcceptList rsOfferTable       = rsOfferTableDefault();
+
+    // Offer for ASK
+    public static AcceptList rsOfferBoolean     = rsOfferBooleanDefault();
+
```

Comment:
So this basically makes all these fields mutable so they can be modified within to the server operators preferred configuration?

Reply:
Yes. Jena has init mechanisms for external code.